kolibri/core/assets/src/core-actions.js

```diff
@@ -54,9 +126,214 @@ function togglemodal(store, bool) {
   }
 }
 
+
+/**
+ * Create models to store logging information
+ * To be called on page load for content renderers
+ */
+function initContentSession(store, Kolibri, channelId, contentId, contentKind) {
```

Comment:
Not sure who you just pinged - but I assume you meant me :)

I talked to @jayoshih about this yesterday. The issue is that if we pull it from the Vuex store, then we are assuming that the place these things are stored in the Vuex store is consistent across all apps that might use the content logging. Referencing the vuex store in the previous way essentially ties this core functionality to the particular way it happens to be instantiated in the learn app.